README.md

```diff
@@ -29,21 +31,39 @@ You may now have to change the owner of your `.docker` directory if the owner is
 chown -R `whoami` ~/.docker
 ```
 
-Create a virtual machine to run the application's containers using the Make task. The default size didn't appear to be large enough so this will create one with an increased disk size:
+Create a virtual machine to run the application's containers using `docker-machine`. The default size didn't appear to be large enough so this will create one with an increased disk size:
```

Comment:
Change to: "The default size isn't large enough"

"didn't appear" is confusing language, it either is, is not, or we don't know what the problem is but this fixes it. We should be clear in this README about which of those it is.